importanize/formatters.py

```diff
@@ -0,0 +1,155 @@
+        """Rename some statement attributes to simplify code."""
+        self.leafs = self.statement.unique_leafs
+        self.stem = self.statement.stem
+        self.comments = self.statement.comments
+        self.string = self.statement.as_string()
+        self.file_artifacts = self.statement.file_artifacts.get('sep', '\n')
+        self.all_comments = (
+            self.comments + list(itertools.chain(
+                *list(map(operator.attrgetter('comments'), self.leafs))
+            ))
+        )
+
+
+class IndentWithTabsFormatter(Formatter):
+    """Default formatter used to organize long imports
+
+    Imports are added one by line preceded by 4 spaces, here's a sample output:
+
+    from other.package.subpackage.module.submodule import (
+        CONSTANT,
+        Klass,
+        bar,
+        foo,
+        rainbows,
+    )
+    """
+
+    def format(self):
+        self.init()
+
+        if len(self.all_comments) == 1:
+            self.string += '  # {}'\
+                .format(' '.join(get_normalized(self.all_comments)))
+
+        if any((len(self.string) > 80 and len(self.leafs) > 1,
+                len(self.all_comments) > 1)):
+            sep = '{}    '.format(self.file_artifacts)
+
+            self.string = 'from {} import ('.format(self.stem)
+
+            if self.comments:
+                self.string += '  # {}'.format(' '.join(
+                    get_normalized(self.comments)
+                ))
+
+            for leaf in self.leafs:
+                self.string += sep
+
+                first_comments = list(filter(
+                    lambda i: i.is_comment_first,
+                    leaf.comments
+                ))
+                if first_comments:
+                    self.string += sep.join(
+                        '# {}'.format(i)
+                        for i in get_normalized(first_comments)
+                    ) + sep
+
+                self.string += '{},'.format(leaf.as_string())
+
+                inline_comments = list(filter(
+                    lambda i: not i.is_comment_first,
+                    leaf.comments
+                ))
+                if inline_comments:
+                    self.string += '  # {}'.format(
+                        ' '.join(get_normalized(inline_comments))
+                    )
+
+            self.string += '{})'.format(self.file_artifacts)
+
+        return self.string
+
+
+class VerticalHangingFormatter(Formatter):
+    """Alternative formatter used to organize long imports
+
+    Imports are added one by line and aligned with the opening parenthesis,
+    here's a sample output:
+
+    from package.subpackage.module.submodule import (CONSTANT,
+                                                     Klass,
+                                                     bar,
+                                                     foo,
+                                                     rainbows,
+                                                     )
```

Comment:
I've seen after that you proposed another syntax in the issue #12 . I will change this.